### PR TITLE
Updates coordinator deployment scripting

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

**Review by theref — APPROVED** [review]

File: scripts/verify_contract.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from ape import networks
+
+etherscan = networks.provider.network.explorer
+etherscan.publish_contract("0xc7590Ba497d1A7AB82c1BD61f7Dcd2685990986e")
```
Comment (derekpierre) [inline scripts/verify_contract.py:4]: Should the contract address be configurable?

Comment (KPrasch) [reply scripts/verify_contract.py:4]: Absolutely.  I couldn't get this working on polygonscan so I stopped short here.

Comment (theref) [reply scripts/verify_contract.py:4]: Yeah there is a bug at the mo. But they have been able to reproduce it and a fix is on the way https://discord.com/channels/922917176040640612/954593641970696292/1109206518286528613

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**